java/gazelle/BUILD.bazel

```diff
@@ -47,7 +47,6 @@ go_test(
         "generate_test.go",
         "resolve_test.go",
     ],
-    data = glob(["testdata/**"]),
```

Comment:
Please add the `# gazelle:exclude testdata` gazelle [directive](https://github.com/bazelbuild/bazel-gazelle#directives) to this file; this prevent running format.sh from adding this back.